src/reporter.ts

```diff
@@ -57,14 +58,17 @@ class RPReporter implements Reporter {
 
   testItems: Map<string, TestItem>;
 
-  suitesAttributes: Array<Attribute[]>;
+  suitesAttributes: Map<string, Array<Attribute>>;
+
+  suitesDescription: Map<string, string>;
 
```

Comment:
Now I think you can create only one map with suites info and hold there attributes, description, etc.